### PR TITLE
adding a check that ssh port is open before running on remote host.

### PR DESCRIPTION
**Pull request by sergey-stratoscale**
+exiting with prettier message on busy hosts

@Stratoscale/systegration PR

### CODE REVIEW

**Comment by adir-stratoscale** [comment]
ship it

File: py/strato/common/log/log2text.py [hunk 1]
```diff
@@ -265,11 +265,18 @@ def tailFile(filePath, n):
...
+        ip = socket.gethostbyname(host)
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        s.settimeout(10)
+        s.connect((ip, 22))
+        s.close()
         return True
-    except:
+    except Exception as e:
+        if e.message == "timed out":
+            print "Connection has timed out"
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: to host.....

File: py/strato/common/log/log2text.py [hunk 2]
```diff
@@ -168,8 +169,14 @@ def follow_generator(istream):
 
 
 def printLog(logFile, formatter, follow, tail):
-    inputStream = sys.stdin if logFile == "-" else universalOpen(logFile, 'r', tail=tail)
-    logTypeConf = formatter._getLogTypeConf(logFile)
+    if not logFile and select.select([sys.stdin,],[],[],0.0)[0]:
```
Comment (lior-stratoscale) [inline py/strato/common/log/log2text.py]: use fileinput instead of select
(see loggingtools/bin/grepjson)

File: py/strato/common/log/log2text.py [hunk 3]
```diff
@@ -258,11 +265,18 @@ def tailFile(filePath, n):
     p = subprocess.Popen(["tail", "-n", str(n), filePath], stdout=subprocess.PIPE)
     return p.stdout
 
-def resolveHostname(host):
+def checkSshConnectivity(host):
     try:
-        socket.gethostbyname(host)
+        ip = socket.gethostbyname(host)
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```
Comment (lior-stratoscale) [inline py/strato/common/log/log2text.py]: please use meaningful names

**Pull request closed without merging**